source/_integrations/google.markdown

```diff
@@ -14,7 +14,7 @@ ha_codeowners:
 ha_integration_type: integration
 ---
 
-The Google integration allows you to connect to your [Google Calendars](https://calendar.google.com) to Home Assistant. The integration adds calendar entities that are shown on the *Calendar* dashboard, and can be used for automations based on any event, or limited to specific matching criteria.
+The *Google Calendars* integration allows you to connect your [Google Calendars](https://calendar.google.com) to Home Assistant. The integration adds calendar entities that are shown on the *Calendar* dashboard, and can be used for automations based on any event, or limited to specific matching criteria.
```

Comment:
The correct name is `Google Calendar` which was fixed in the next release.

Reply:
Yes, that's fine -- can you just adjust for the merge conflicts?

Reply:
Sure, I'll do - thanks.

Reply:
Done, waiting for approval.